Sources/Controllers/2.0/OASuperViewController.m

```diff
@@ -46,13 +46,26 @@ - (BOOL)isModal
 }
 
 - (IBAction) backButtonClicked:(id)sender
+{
+    [self dismissViewController];
+}
+
+- (void) dismissViewController
 {
     if ([self isModal])
         [self dismissViewControllerAnimated:YES completion:nil];
     else
         [self.navigationController popViewControllerAnimated:YES];
 }
```

Comment:
I thought it was wrong that many places in the code used IBAction:(nil) instead of a normal function. So I added a separate function for that. IBAction is now used only by buttons.